cdap-common/src/main/java/co/cask/cdap/common/utils/ApplicationBundler.java

```diff
@@ -171,7 +172,7 @@ public String apply(Class<?> input) {
 
     // Record the set of classpath URL that are already added to the jar
     final Set<URL> seensClassPaths = Sets.newHashSet();
-    Dependencies.findClassDependencies(classLoader, new Dependencies.ClassAcceptor() {
+    Dependencies.findClassDependencies(classLoader, new ClassAcceptor() {
```

Comment:
You can actually remove this class and replace usage with the ApplicationBundler in Twill since TWILL-112 is already resolved for twill-0.6.0